projects/plugins/boost/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "jetpack-boost",
-	"version": "2.0.0-alpha",
+	"version": "2.0.0-beta",
```

Comment:
This doesn't seem right, since we're releasing 1.4.1-beta.

Reply:
Thanks; I missed this one. Fixed it, and `composer.json` in 2fe9295108.